backend/dataall/core/environment/api/resolvers.py

```diff
@@ -28,7 +28,14 @@
     NamingConventionPattern,
 )
 from dataall.core.organizations.api.resolvers import Context, exceptions, get_organization
-from dataall.core.permissions.services import core_permissions
+from dataall.core.permissions.services.permissions_constants.environment_permissions import (
```

Comment:
We should place the permissions of a module inside the module itself as we do in any other module. For example, take notebooks, we define `notebook_permissions` in `modules/notebooks/service/notebooks_permissions.py`. In the same way environment permissions should be defined in `core/environments/service/environment_permissions.py` And the same applies to other core modules in this last commit